app/models/views/application_overview.rb

```diff
@@ -0,0 +1,53 @@
+module Views
+  class ApplicationOverview
+    attr_reader :application
+
+    APPLICATION_ATTRS = %i[date_of_birth full_name ni_number
+                           date_received form_name amount_to_pay]
+
+    delegate(*APPLICATION_ATTRS, to: :application)
+
+    def initialize(application)
+      @application = application
+    end
+
+    def status
+      @application.married? ? 'Married' : 'Single'
+    end
+
+    def jurisdiction
+      @application.jurisdiction.name
+    end
+
+    def fee
+      "£#{@application.fee.round}"
+    end
+
+    def number_of_children
+      @application.children
+    end
+
+    def total_monthly_income
+      "£#{@application.income.round}"
+    end
+
+    def income
+      {
+        'full' => I18n.t('activerecord.attributes.application.summary.passed'),
+        'part' => I18n.t('activerecord.attributes.application.summary.passed'),
+        'none' => I18n.t('activerecord.attributes.application.summary.failed')
+      }[@application.application_outcome]
+    end
+
+    def savings
+      {
+        'true' => I18n.t('activerecord.attributes.application.summary.passed'),
+        'false' => I18n.t('activerecord.attributes.application.summary.failed')
+      }[@application.savings_investment_valid?.to_s]
+    end
```

Comment:
This is a small suggestion. See what you think.

This method **savings** & the one above **income** potentially could work as:

```ruby
    def income
      format_locale @application.application_outcome
    end

    def savings
      format_locale @application.savings_investment_valid?.to_s
    end

    private

    def format_locale(value)
      prefix = 'activerecord.attributes.application.summary.'
      I18n.t([prefix, value].join)
    end
```

And then in the locale file add the appropriate keys:

```
activerecord.attributes.application.summary.(part|full|none|false|true)
```

What do we gain with this change? 
Looking at it now, the repetition would be moved from the codebase to the locale keys.

Also, now that I've thought about it a bit more, this might not work once we move away from the monolithic application model... The keys might have to be stored under a different hierarchy...

See what you think.